salt/states/virt.py

```diff
@@ -567,21 +569,21 @@ def running(
         .. versionadded:: 3000
 
     :param boot:
-        Specifies kernel for the virtual machine, as well as boot parameters
-        for the virtual machine. This is an optionl parameter, and all of the
-        keys are optional within the dictionary. If a remote path is provided
-        to kernel or initrd, salt will handle the downloading of the specified
-        remote fild, and will modify the XML accordingly.
+        Boot parameter definition is documented in the :func:`init` function. To update any
+        boot parameters, specify the new path for each. To delete any boot parameters,
+        pass a None object, for instance: 'kernel': None.
 
         .. code-block:: python
 
             {
-                'kernel': '/root/f8-i386-vmlinuz',
-                'initrd': '/root/f8-i386-initrd',
-                'cmdline': 'console=ttyS0 ks=http://example.com/f8-i386/os/'
+                'kernel': '/root/new-vmlinuz',
+                'initrd': '/root/new-initrd',
+                'cmdline': 'console=ttyS0 ks=http://example.com/f8-i386/os/',
+                'loader': '/usr/share/new/OVMF_CODE.fd',
+                'nvram': '/usr/share/new/OVMF_VARS.ms.fd'
             }
 
-        .. versionadded:: 3000
+        .. versionadded:: sodium
```

Comment:
Same as for the `virt.defined` state comment. But keep the `versionadded` to `3000` since that's when the parameter first appeared